src/main/java/core/basesyntax/IsoscelesTrapezoidSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoidSupplier {

```

Comment:
The class 'IsoscelesTrapezoidSupplier' should implement or extend the 'FigureSupplier' interface or class to maintain consistency and to follow the polymorphism principle.